test/api/assignments_test.rb

```diff
@@ -130,16 +130,16 @@ def test_notify_team_members_about_submission
   end
 
   def test_api_rejects_duplicates
-    Attempt.new(alice, Iteration.new({'THE CODE' => 'ruby/one/code.rb'}, 
+    Attempt.new(alice, Iteration.new({"THE CODE" => "ruby/one/code.rb"}, 
     {track: 'ruby', slug: 'one' })).save
     Notify.stub(:everyone, nil) do
       Xapi.stub(:exists?, true) do
         solution = { "THE CODE" => "ruby/one/code.rb" }
-        post '/user/assignments', { key: alice.key, code: 'THE CODE', path: 'ruby/one/code.rb', 
-          'language' => 'ruby', 'problem' => 'one', solution: solution }.to_json
+        post '/user/assignments', { key: alice.key, code: "THE CODE", path: "ruby/one/code.rb",
```

Comment:
Line is too long. [95/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.